community/import-tool/src/test/java/org/neo4j/tooling/ImportToolTest.java

```diff
@@ -920,12 +945,22 @@ private void writeNodeData( PrintStream writer, Configuration config, List<Strin
         {
             if ( linePredicate.accept( i ) )
             {
-                writer.println( nodeIds.get( i ) + delimiter + randomName() +
-                                delimiter + randomLabels( arrayDelimiter ) );
+                writer.println( getLine( nodeIds.get( i ), delimiter, arrayDelimiter, extraColumns ));
             }
         }
     }
 
+    private String getLine(String nodeId, char delimiter, char arrayDelimiter, int columnCount) {
```

Comment:
last param called columnCount while all others method in a call chain use `extraColumns` can we align those?